typescript/hardhat/src/TestAbacusDeploy.ts

```diff
@@ -112,43 +78,39 @@ export class TestAbacusDeploy extends TestDeploy<
 
     const inboxFactory = new TestInbox__factory(signer);
     const inboxes: Record<types.Domain, TestInbox> = {};
-    const inboxDeploys = remotes.map(async (remote) => {
-      const inboxMultisigValidatorManager =
-        inboxMultisigValidatorManagers[remote];
+
+    // this.remotes reads this.instances which has not yet been set.
+    const remotes = Object.keys(this.config.signer).map((d) => parseInt(d));
+    const deploys = remotes.map(async (remote) => {
       const inbox = await inboxFactory.deploy(domain);
+      // Inbox will require the validator manager to be a contract. We don't
+      // actually make use of the validator manager, so we just pass in the
+      // upgradeBeaconController as the validator manager to satisfy the contract
```

Comment:
It looks like this comment is wrong, we're passing the xcm. Consider passing the ubc for conssitency